torchvision/ops/misc.py

```diff
@@ -131,9 +132,12 @@ def __init__(
         activation_layer: Optional[Callable[..., torch.nn.Module]] = torch.nn.ReLU,
         dilation: int = 1,
         inplace: bool = True,
+        bias: Optional[bool] = None,
     ) -> None:
         if padding is None:
             padding = (kernel_size - 1) // 2 * dilation
+        if bias is None:
```

Comment:
These two statements could be removed by adding
```
bias=bias or norm_layer is None
```
on line 150(given bias is bool and defaults to False).

Reply:
I don't think this is equivalent: This would override bias whenever bias is False. We only want to override bias when it's None.

Reply:
I see, my understanding was wrong. I thought if bias is False we default back to `norm_layer is None`